packages/iTwinUI-react/src/core/ErrorPage/ErrorPage.test.tsx

```diff
@@ -86,11 +142,46 @@ describe(ErrorPage, () => {
   });
 
   const customTests = [
+    {
+      errorType: '300',
+      errorName: '300 error',
+      errorMessage: 'Page has multiple choices',
+    },
     {
       errorType: '301',
       errorName: '301 error',
       errorMessage: 'Page moved permanently',
     },
+    {
+      errorType: '302',
+      errorName: '302 error',
+      errorMessage: 'Page not found',
+    },
+    {
+      errorType: '303',
+      errorName: '303 error',
+      errorMessage: 'Page moved temporarily',
+    },
+    {
+      errorType: '304',
+      errorName: '304 error',
+      errorMessage: 'Page not modified',
+    },
+    {
+      errorType: '305',
+      errorName: '305 error',
+      errorMessage: 'Page use proxy',
+    },
+    {
+      errorType: '307',
+      errorName: '307 error',
+      errorMessage: 'Page moved permanently',
```

Comment:
The [MDN article](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#:~:text=The%20server%20sends,the%20second%20request.) says that 307 is for a temporary redirect instead of a permanent move.

Reply:
Resolved